lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -208,7 +215,7 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
         return selections.filter(opt => !existingNode.some((node) => node.id === opt.id));
     }
 
-    downloadContent(file: Node): void {
+    onDownloadFile(file: Node): void {
```

Comment:
should be `onDownloadNode(node: Node)` as you deal with node, not file

Reply:
done.